File: LOO 2°Parcial/Simulador de Vehiculos CSharp/Simulador de Vehiculos CSharp/Remolque.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Simulador_de_Vehiculos_CSharp
{
    class Remolque 
    {
        protected double pMax, alto, ancho, largo;
        Remolque(double pMax, double alto, double ancho, double largo)
        {
            this.pMax = pMax;
            this.alto = alto;
            this.ancho = ancho;
            this.largo = largo;
        }
        /*public override String toString()
        {
            return ("\nRemolque" + "\nAltura: " + this.alto + "\nAncho: " + this.ancho+ "\nLargo: " + this.largo+ "\nPeso maximo" + this.pMax);
        }*/
    }
}

File: LOO 2°Parcial/Java/Remolque.java
public class Remolque{
  public double pMax, alto, ancho, largo;
  Remolque(double pMax, double alto, double ancho, double largo){
    this.pMax = pMax;
    this.alto = alto;
    this.ancho = ancho;
    this.largo = largo;
  }

  @Override
  public String toString(){
    return "\n///REMOLQUE/// \nEl remolque tiene las siguientes medidas: \nAltura: " + this.alto + "\nAncho: " + this.ancho + "\nLargo: " + this.largo + "\nY puede cargar: " + this.pMax;
  }
}